frontend/device/sdl/device.lua

```diff
@@ -59,6 +59,16 @@ function Device:init()
     Generic.init(self)
 end
 
+function Device:setTime(hour, min)
```

Comment:
Sorry, I meant calls to Device:setTime() like in `if Device:setTime() then`, not this function here. Like this:

```
> Device = {}
> function Device:simulateSuspend() end
> Device:simulateSuspend()
> Device:setTime()
stdin:1: attempt to call method 'setTime' (a nil value)
stack traceback:
	stdin:1: in main chunk
	[C]: in ?
```

So you'd need something like `function Device:setTime() end` or `function Device:setTime() return false end` in Device unless I'm overlooking something.